airbyte-integrations/connectors/source-stripe/source_stripe/schemas/invoices.json

```diff
@@ -144,13 +144,13 @@
           "type": ["null", "integer"]
         },
         "invoice": {
-          "type": ["null", "integer"]
+          "type": ["null", "string"]
         },
         "invoice_item": {
-          "type": ["null", "integer"]
+          "type": ["null", "string"]
         },
         "promotion_code": {
-          "type": ["null", "integer"]
+          "type": ["null", "string"]
```

Comment:
It looks like promotion code was the cause of the failure, are we sure we need to modify the other fields, too?

Reply:
I believe so, the other fields should also match expected type.

Reply:
Okay cool, I can see from Stripe's documentation that `redeem_by` is a timestamp and `inovice_item` is a string, and from the error `promotion_code` also looks to be string. I'm not specifically seeing `invoice` but from the patterns in the documentation I believe it would be too.